### PR TITLE
Handle empty rule

### PR DESCRIPTION
**Pull request by skrypka**
@kmike 
Hi,
Simple fix for the second bug in issue https://github.com/scrapinghub/splash/issues/399

### CODE REVIEW

File: adblockparser/parser.py [hunk 1]
```diff
@@ -291,7 +291,7 @@ def __init__(self, rules, supported_options=None, skip_unsupported_rules=True,
         self.rules = [
             r for r in (
                 r if isinstance(r, rule_cls) else rule_cls(r)
-                for r in rules
+                for r in rules if r
```
Comment (kmike) [inline adblockparser/parser.py]: I think we should strip whitespaces before checking if rule text is empty: AdblockRule class strips whitespaces, it makes sense to be consistent with it.

**Comment by kmike** [comment]
Awesome, thanks @skrypka!

**Comment by skrypka** [comment]
@kmike 
Yeah, agree. I just didn't want to create extra complexity by stripping and handling AdblockRule class.
But after your message I find out an even better way to handle the situation... just marking a rule as a comment.

**Comment by kmike** [comment]
Thanks @skrypka! Test failure is unrelated.

**Pull request merged**